index.html

```diff
@@ -194,46 +293,76 @@ <h3 class="aboutme-techs-title">Skills</h3>
+        </div>
+       <div class="contact-card">
+
+        <div class="contact-title-d">
+          <h2 class="contact-title">Get Started</h2>
+          <p class="contact-para">I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.</p>
+        </div>  
+          <form action="https://formspree.io/f/xqkovbew"
+            method="POST" class="contact">
+          <div class="input-d1">
+            <label for="name">Your name</label>
+            <input type="text" id="fname" maxlength="30"  placeholder="Enter your name" required>
+          </div>
+          <div class="input-d2">
+            <label for="email">Your email</label>
+            <input type="email" id="email" placeholder="your email" required>
+          </div> 
+
+          <div class="input-d3">
+            <label for="message">Your message</label>
+            <textarea name="your message" id="" cols="30" rows="10" 
+              placeholder="Enter your message here..." maxlength="500" required></textarea>
+            <button type="submit" value="Start Collaboration" class="form-btn">Start Collaboration</button>
+          </div>  
+          </form>
+          
+       </div>
+
+      <div class="contact-logo2-d">
+        <img  class="contact-img" src="./images/image-geometry_7.png" alt="logo desktop2">
+      </div>
+
+
   </div>
-    <div class="contact-footer">
-      <ul class="get">
-         <li><h3>Get my resume</h3></li>
-         <li><i class="fa fa-external-link"></i></li>
-      </ul>
-      
-      
-      <ul class="contact-social"> 
-        <li><a href=""><i class="fa fa-stack-overflow fa-inverse"></i></a></li>
-        <li><a href=""><i class="fa fa-github fa-inverse "></i></a></li>
-        <li><a href=""><i class="fa fa-twitter fa-inverse"></i></a></li>
-        <li><a href=""><i class="fa fa-linkedin fa-inverse"></i></a></li>
-        <li><a href=""><i class="fa fa-medium fa-inverse"></i></a></li>
-        <li><a href=""><i class="fa fa-angellist fa-inverse"></i></a></li>
-      </ul>
+
+      <div class="contact-footer">
+          <ul class="get">
+            <li><h3>Get my resume</h3></li>
+            <li><i class="fa fa-external-link remove"></i></li>
+            <li class="get3"><span> Get my  CV</span> <img src="./images/ic_download.png" alt="ic_download"></li>
+          </ul>
+         
+        
+        <ul class="contact-social"> 
+          <li><a href=""><i class="fa fa-stack-overflow fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-github fa-inverse "></i></a></li>
+          <li><a href=""><i class="fa fa-twitter fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-linkedin fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-medium fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-angellist fa-inverse"></i></a></li>
+        </ul>
```

Comment:
As suggested by the previous reviewer,  Kindly correct the size and the distance between the inputs to be similar to the Figma design 

your design :
![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/20537712/7d1448b6-a41d-47b6-bdc3-ad5b4dc06fbc)